fetch.bs

```diff
@@ -6894,7 +6894,7 @@ constructor steps are:
    <dd><var>request</var>'s <a for=request>priority</a>.
 
    <dt><a for=request>origin</a>
-   <dd>"<code>client</code>".
+   <dd><var>request</var>'s <a for=request>origin</a>.
```

Comment:
Please add a note here about how particular the situation is where this is relevant. Probably with `<span class=note>` as we don't seem to have paragraphs here.

Reply:
Done.